denis/configure.py

```diff
@@ -49,23 +57,26 @@ def add_final(parser, required=True):
     subparser_name = kwargs.pop('command')
     # Get a handle to the function with the desired name
     subparser_func = globals()[subparser_name]
-    # Call the desired subparser with the remaining flags and their values as kwargs
+    # Call the desired subparser with the remaining flags/values as kwargs
```

Comment:
This hunk should actually go on the previous commit. You can fix it with git rebase. I would just copy this version of the comment and then go back to the first commit change the line there and amend and then when you continue rebasing it should recognize what happened and just drop the hunk from this second commit